benefits/static/css/styles.css

```diff
@@ -32,12 +28,16 @@
   --h1-text-align: left;
   --h2-font-size: calc(24rem / 16);
   --h3-font-size: calc(20rem / 16);
+  --main-content-min-height: calc(100vh - 182px);
+  /* 182px = Header Height (80px) + (Footer Link (50px) * Number of Links (2)) + Underline Height (2px) */
 }
 
 @media (min-width: 992px) {
   :root {
     --h1-font-size: calc(35rem / 16);
     --h1-text-align: center;
+    --main-content-min-height: calc(100vh - 130px);
+    /* 130px = Header Height (80px) + Footer Height (50px) */
```

Comment:
This line was initially inside the `@media (min-width: 767px)` media query at b35e375d8a1b14dfab5a6e925da26f44494413dd, but then the media query moved in https://github.com/cal-itp/benefits/pull/1014/commits/10cbca03a5230dffb5528e5a7890c86da1031827#diff-744df555aee38173a0b0868baa7befa720e6a28f80747523e74e724c126cd381L46-L47 without taking this line. I can't tell what the intended change was, so could you clarify?

Reply:
https://github.com/cal-itp/benefits/pull/1014/commits/f9b605fbd115639ee5c4a4301ae590ae951378d7

Footer breakpoint is 992px (can be changed if Design wants to change it)

I want Design to test the footer on dev-benefits.calitp.org on real devices before we change the breakpoint though.